test/parallel/test-readline-interface.js

```diff
@@ -371,6 +371,37 @@ function isWarned(emitter) {
     }));
   }
 
+  // constructor throws if historySize is not a positive number
+  {
+    const fi = new FakeInput();
+    assert.throws(function() {
+      readline.createInterface({
+        input: fi, historySize: 'not a number'
+      });
+    }, common.expectsError({
+      type: RangeError,
+      code: 'ERR_INVALID_OPT_VALUE'
+    }));
```

Comment:
It would be nice to use `common.expectsError(throwingFn, errorObj)` instead of `assert.throws(throwingFn, common.expectsError(errorObj))`.